<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureVmDialog</class>
 <widget class="QDialog" name="ConfigureVmDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>455</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Configure VM</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="_locationLineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="_typeLabel">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="_nameLineEdit"/>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QLineEdit" name="_templateLineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QTreeWidget" name="_componentsTreeWidget">
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="_architectureLabel">
     <property name="text">
      <string>Architecture:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLineEdit" name="_architectureLineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="_locationLabel">
     <property name="text">
      <string>Location:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QGroupBox" name="_propertiesGroupBox">
     <property name="title">
      <string>Properties</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="0" column="1">
       <widget class="QLineEdit" name="_componentNameLineEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="_componentNameLabel">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QScrollArea" name="_componentEditorsScrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>240</width>
           <height>187</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="_addComponentPushButton">
     <property name="text">
      <string>Add component</string>
     </property>
     <property name="icon">
      <iconset resource="emuone.qrc">
       <normaloff>:/emuone/Images/Actions/Add.png</normaloff>:/emuone/Images/Actions/Add.png</iconset>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="_removeComponentPushButton">
     <property name="text">
      <string>Remove component</string>
     </property>
     <property name="icon">
      <iconset resource="emuone.qrc">
       <normaloff>:/emuone/Images/Actions/Remove.png</normaloff>:/emuone/Images/Actions/Remove.png</iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="_templateLabel">
     <property name="text">
      <string>Template:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLineEdit" name="_typeLineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="_nameLabel">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="emuone.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigureVmDialog</receiver>
   <slot>_accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigureVmDialog</receiver>
   <slot>_reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_addComponentPushButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureVmDialog</receiver>
   <slot>_addComponentPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_removeComponentPushButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureVmDialog</receiver>
   <slot>_removeComponentPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_accept()</slot>
  <slot>_reject()</slot>
  <slot>_addComponentPushButtonClicked()</slot>
  <slot>_removeComponentPushButtonClicked()</slot>
 </slots>
</ui>
